cypress/integration/chi-custom-elements/drawer.spec.js

```diff
@@ -0,0 +1,113 @@
+/// <reference types="Cypress" />
+
+describe('Drawer', function() {
+
+  beforeEach(()=>{
+    cy.fixture('chidata.json').as('chidata');
+  });
+
+  it('Drawer custom element should initiate a Chi drawer component. ', function() {
```

Comment:
I usually like ES6 arrow functions rather than the function keyword and I'm using that in my code, so just to be consistent let's pick one, I really don't know what ES standards we use in Chi so just putting a thought there.

Reply:
@srinath-koneru, Yeah! You're right. I also prefer the arrow ones. But as in the examples in the documentation they used the old way, I did the same. Now, when trying to implement this requested change, I realized the icons tests began not to pass. It is related to the _Avoiding the use of this_ section in https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html#Sharing-Context
There's a workaround using an asynchronous function that will make the tests a little more difficult to write. 
![change](https://user-images.githubusercontent.com/43439749/58246316-028e0780-7d57-11e9-8e80-134a2100f8e6.png)

@srinath-koneru , @cristb Which option do you prefer? My vote goes to keep function keyword.

Reply:
I love arrow functions as well. However, in order to simplify cypress code, I would use the old way.

Reply:
TY @cristb , @srinath-koneru , what is your opinion?

Reply:
either one is fine I'm usually inclined to new ES standards but either way I want our code to be consistent, I'll change my PR accordingly.